configs/config.toml.SAMPLE

```diff
@@ -51,6 +51,7 @@
 # Addresses of contracts deployed on ethereum blockchain.
 [ethereum.ContractAddresses]
   BondedECDSAKeepFactory = "0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC"
+  TBTCSystem = "0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF"
```

Comment:
… Yes…

The main reason I didn't do this originally was to avoid dropping the whole tBTC-specific section in the config, but since we're talking about moving all extension config under chains, I think doing it now makes sense.

Reply:
Done.